node/builder.go

```diff
@@ -283,6 +289,14 @@ func ConfigCommon(cfg *config.Common) Option {
 			Override(new(lotus_dtypes.BootstrapPeers), modules.ConfigBootstrap(cfg.Libp2p.BootstrapPeers)),
 		),
 
+		Override(new(network.ResourceManager), lp2p.ResourceManager(cfg.Libp2p.ConnMgrHigh)),
```

Comment:
You also need the ResourceManagerKey thing from lotus to actually pass the resource manager option to libp2p